src/features/annotation-thread/AnnotationThread.scss

```diff
@@ -0,0 +1,10 @@
+@import 'src/elements/common/variables';
+
+$annotationThreadWidth: 340px;
+$annotationThreadHeight: 600px;
```

Comment:
These could be SUIT styled: `$AnnotationThread-width`

https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#variables